### ISSUE TITLE
v3 spec: no compression?

### ISSUE THREAD

**Issue by davidbrochart** [zarr-developers/zarr-specs#93]
It looks like an array does not necessarily have a compressor ("An array may be associated with a compressor").
How is it encoded in the array metadata? If it is specified by the absence of a `compressor` key, the spec should be explicit about it.

**Comment by davidbrochart** [comment]
Maybe we can have the following in the array metadata:
```json
    "compressor": {
        "codec": "https://purl.org/zarr/spec/codec/binary/1.0",
        "configuration": {}
    },

```

**Comment by jakirkham** [comment]
Based on discussion, we decided to simply not have a `compressor` key if the data is not compressed.

**Comment by Carreau** [comment]
Closed by #94
